tests/trainer/legacy_deprecate_flow_log_tests/test_eval_loop_dict_return.py

```diff
@@ -125,6 +126,9 @@ def test_validation_step_dict_return(tmpdir):
     Test that val step can return a dict with all the expected keys and they end up
     in the correct place
     """
+
+    os.environ['PL_DEV_DEBUG'] = '0'
```

Comment:
what is wrong if the debugger is used?

Reply:
It was adding `debug_epoch` key.

Reply:
yes, but it kind of deactivating, right?
then it shall be rather deactivated after each usage...